### ISSUE TITLE
incremental save

### ISSUE THREAD

**Issue by mkrierQape** [dsteinkopf/backup-all-mysql#16]
Hello, 

it possible to modify the script for  incremental save ? currently every time is script is running its deleting previous save, can you make a options for saving the file with the current date ? Something like :  $DBDUMPSDIR/mysqldump_$db_YYYY_MM_DD_HH_MM.sql.bz2 

Thank

**Comment by dsteinkopf** [comment]
Yes, this would be possible, of course. The reason why I didn't do this is the "unix idea of orthogonal tools": backup-all-mysql only does the dumps on a regular basis. A different tool (backuppc in my case) moves the backup files to a different media and does versioning. - With that in mind, it would not be a good idea to create a different filename for every backup.

If you really would like to implement such a filename scheme (as suggested by you), feel free to implement that and send a PR. But please keep an option to always get the same filename - just as it's currently done. - Maybe a configurable filename pattern with a default that's the current filename ("mysqldump.sql").

(Sorry for answering so late - I was on holiday and then I "forgot" this notification email.)

**Comment by Jogai** [comment]
Reminder for when I get back to this (or for anyone that wants to do this ;), 

`mkdir -p "${date +%Y/%m/%d/%H/%M/%S}"`
